packages/manager/src/managers/sliceTemplateLibrary/SliceTemplateLibraryManager.ts

```diff
@@ -0,0 +1,179 @@
+import * as t from "io-ts";
+import { HookError } from "@slicemachine/plugin-kit";
+import { BaseManager } from "../BaseManager";
+import { DecodeError } from "../../lib/DecodeError";
+import { assertPluginsInitialized } from "../../lib/assertPluginsInitialized";
+import { decodeHookResult } from "../../lib/decodeHookResult";
+import { SharedSlice } from "@prismicio/types-internal/lib/customtypes";
+import { SharedSliceContent } from "@prismicio/types-internal/lib/content";
+
+type SliceTemplateLibraryManagerReadReturnType = {
+	templates: {
+		model: SharedSlice;
+		screenshots: Record<string, Buffer>;
+	}[];
+	errors: (DecodeError | HookError)[];
+};
+
+type SliceTemplateLibraryManagerCreateReturnType = {
+	data?: {
+		sliceIds: string[];
+	};
+	errors: (DecodeError | HookError)[];
+};
+
+type SliceTemplateLibraryReadData = {
+	templateIds?: string[];
+};
+
+type SliceTemplateLibraryCreateData = SliceTemplateLibraryReadData;
+
+const readHookCodec = t.type({
+	templates: t.array(
+		t.type({
+			model: SharedSlice,
+			componentContents: t.string,
+			mocks: t.array(SharedSliceContent),
+			screenshots: t.record(t.string, t.any),
+		}),
+	),
+});
+
+export class SliceTemplateLibraryManager extends BaseManager {
+	async read(
```

Comment:
To keep consistent with the other managers, could these methods be explicit about what it is reading? For example, `readLibrary()` or `readSliceTemplateLibrary()`.

Being explicit helps us add other read-related methods, like `readSliceLibraryTemplateMetadata` (if we needed something like that).

Reply:
Done

Reply:
- readLibrary
- createSlices